dask_planner/src/sql/logical/create_model.rs

```diff
@@ -80,6 +82,11 @@ impl PyCreateModel {
         Ok(self.create_model.input.clone().into())
     }
 
+    #[pyo3(name = "getSchemaName")]
+    fn get_schema_name(&self) -> PyResult<Option<String>> {
+        Ok(self.create_model.schema_name.clone())
+    }
+
```

Comment:
I think this is not possible today, but do you happen to know if there's a way we could have a common utility or template that could be share across classes. 

I'm assuming it might not be straightforward since these are all pyo3 classes and the variable name (`create_model` and so on) wouldn't match the class name of the caller.

Reply:
Would need to do some digging to see (cc @andygrove in case there's an obvious solution I'm missing here), but as long as the attribute the schema name is stored under is consistent we could probably consolidate this somewhat with a macro?